microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/get-started.md

```diff
@@ -0,0 +1,366 @@
+description: Step-by-step tutorial about opening and closing DevTools and using it to modify CSS and debug JavaScript.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 10/06/2022
+---
+# Get started using the DevTools extension for Visual Studio Code
+
+This is a step-by-step tutorial about opening and closing DevTools and using it to modify CSS and debug JavaScript.  Follow the steps here from top to bottom for a general introduction and to ensure that your machine is set up to use DevTools.
+
+
+<!-- ====================================================================== -->
+## Step 1: Install DevTools and prerequisites
+
+1. Do the steps in [Installing the DevTools extension for Visual Studio Code](./install.md), and then continue below.
+
+
+<!-- ====================================================================== -->
+## Step 2: Start DevTools by clicking the Launch Instance button for the default page
+
+This approach opens the DevTools tabs in non-debug mode and is useful when you don't want to use the Debug features of Visual Studio Code.  Supported scenarios for this way of opening DevTools:
+
+*  You don't have to have a folder open in Visual Studio Code if you use a file path; you can still use CSS mirror editing to edit your CSS source files.
+
+*  If you don't have a folder open and you want to inspect a page that's specified using a URL.
+
+*  If you don't have a folder open and you want to experiment with changing CSS on a page that's specified using a URL, without using CSS mirror editing.
+
+Clicking the **Launch Instance** button doesn't go directly to your file; you have to paste a file path or URL.  This approach is prominent in the UI, and you need to know how to close instances of DevTools that have been opened this way.  You should also know how to edit the Success page, so you can get started.
+
+   **Open the DevTools tabs:**
+
+1. In Visual Studio, select **File** > **New Window**.  Initially, no folder is opened.
+
+1. In Activity Bar, click **Microsoft Edge Tools** (![Microsoft Edge Tools icon](./get-started-images/microsoft-edge-tools-icon.png)).  The **Microsoft Edge Tools** Side Bar opens.
+
+1. Click the **Launch Instance** button.  The **Edge DevTools** tab opens, and the **Edge DevTools: Browser** tab opens, displaying the default, Success page.  In the address bar of the DevTools browser, there is a `file:///` path (rather than a URL), such as `file:///C:/Users/myusername/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-2.1.1/out/startpage/index.html`.
+
+   The Debug toolbar doesn't open, **Debug Console** doesn't open at bottom, and the **Run and Debug** Side Bar with **Watch** pane doesn't open.  This indicates that Visual Studio Code is not in Debug mode.
+
+   You can modify CSS of the local files, and you can enter local file paths or localhost URLs in the address bar and interact with local web app pages.
+
+   **Open a folder:**
```

Comment:
Did you really intend to indent these bolded section titles? Since they're indented, the numbered list items don't re-start at 1 after the titles. Maybe that's on purpose, maybe not, I just couldn't tell.

Reply:
by design, though quite possible to switch them to h4s